src/views/Main/Catalog/Components/PermissionsPanel.js

```diff
@@ -101,10 +101,14 @@ export class PermissionsPanel extends React.Component {
 
   loadAcls = (updateSuccess) => {
     if (this.props.propertyTypeName) {
-      // PermissionsApi.getOwnerAclsForPropertyTypesInEntitySet(this.props.entitySetName)
-      // .then((acls) => {
-      //   console.log(acls);
-      // })
+      const fqn = {
```

Comment:
question for you. would your life be easier if you had something like `new FullyQualifiedName(...)`?